cli/cmd/vuln_host_show_assessment.go

```diff
@@ -559,3 +595,16 @@ func sortVulnHosts(slice []api.VulnerabilityHost) {
 		})
 	}
 }
+
+func checkAgentlessCloudAccount() bool {
+	resp, err := cli.LwApi.V2.CloudAccounts.List()
+	if err != nil {
+		return false
+	}
+	for _, ca := range resp.Data {
+		if strings.Contains(ca.Type, "Sidekick") {
```

Comment:
If we actually want to check for an integration, let's make sure to use the constant defined at the `api` package instead of hardcoding a string here. (not that the integrations type will change but, that's why we define constants, right? 😅 )